python/tests/src/tests/unit/test_mapping.py

```diff
@@ -29,7 +29,7 @@ def test_mapping_host_star_error():
     errors = ir.aconf.errors["bad-mapping.default.1"]
     assert len(errors) == 1, f"Expected 1 error but got {len(errors)}"
 
-    assert errors[0]["ok"] == False
+    assert errors[0]["ok"] is False
```

Comment:
Ew. Again, not gonna scream if we don't have a way to tweak `ruff` for this, but while I can vaguely get behind `is None`, `is False` makes me cringe. 😂

Reply:
In all technicality that _should_ be `assert not errors[0]["ok"]` which (arguably) isn't much better 🙃